config/formats.js

```diff
@@ -257,7 +259,9 @@ exports.Formats = [
 	},
 	{
 		name: "[Gen 7] Random Doubles Battle",
-
+      desc: [ 
```

Comment:
this can all be one one line; also, you seem to have an extra space before `desc`. There should also be a blank line below this:

```javascript
		desc: ["&bullet; <a href=\"http://www.smogon.com/forums/threads/3601525/\">Sets and Suggestions</a>"],

		mod: 'gen7',
```

Reply:
It's not an extra space, it's indenting with spaces instead of tabs, which interact differently with diff visualization, hence it appearing to be an extra space.

Reply:
o